lib/charms/opensearch/v0/opensearch_relation_peer_cluster.py

```diff
@@ -269,6 +286,88 @@ def _on_peer_cluster_relation_departed(self, event: RelationDepartedEvent) -> No
             orchestrators.delete("failover")
             self.charm.peers_data.put_object(Scope.APP, "orchestrators", orchestrators.to_dict())
 
+    def _should_promote_failover(self) -> bool:
+        """Check if majority of related apps are disconnected from main orhcestrator"""
```

Comment:
Perhaps for further clarity?
```suggestion
    def _should_promote_failover_to_main(self) -> bool:
        """Check if majority of related apps are disconnected from main orchestrator"""
```

Reply:
Done 👍